modules/lotamePanoramaIdSystem.js

```diff
@@ -68,10 +75,10 @@ function setProfileId(profileId) {
  */
 function getProfileId() {
   let profileId;
-  if (storage.cookiesAreEnabled()) {
+  if (cookiesAreEnabled(false)) {
     profileId = storage.getCookie(KEY_PROFILE, undefined);
   }
-  if (!profileId && storage.hasLocalStorage()) {
+  if (!profileId && localStorageIsEnabled(false)) {
```

Comment:
for code legibility - i think it would be nice to define two constants HONOR_CONFIG=true and DO_NOT_HONOR_CONFIG=false and use those constants because `localStorageIsEnabled(false)` confused me quite a bit on first read through - i thought it and several of the others below were saying the opposite of what they were meaning.

Reply:
It's a parameter, not a value, but I see your point.